src/dmd/dsymbolsem.d

```diff
@@ -6076,6 +6075,29 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
             }
         }
 
+        version (none) // Controversial - see Bugzilla 519
```

Comment:
So are we going to do it, or not?

Reply:
Seems like this decision is the only thing holding this PR of. How about splitting it of into a separate PR?